app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -53,7 +49,7 @@ class MainFragment : Fragment() {
             prepTakePhoto()
         }
 
-        button3.setOnClickListener {
+        getRandomPuzzleButton.setOnClickListener {
```

Comment:
Good idea.  This is more descriptive.  Could also make it btnRandomPuzzle, so it's not confused with a traditional getter method.